.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "node"
+  - "7"
+  - "6"
+  - "5"
```

Comment:
Umm...I am not sure what does the line 4,5,6 do here? Can you please elaborate this? if these not required, then please remove them.